pkg/manager/member/dm_worker_member_manager.go

```diff
@@ -507,9 +522,9 @@ func getNewWorkerSetForDMCluster(dc *v1alpha1.DMCluster, cm *corev1.ConfigMap) (
 				},
 			},
 			ServiceName:         controller.DMWorkerPeerMemberName(dcName),
-			PodManagementPolicy: apps.ParallelPodManagement,
+			PodManagementPolicy: baseWorkerSpec.PodManagementPolicy(),
 			UpdateStrategy: apps.StatefulSetUpdateStrategy{
-				Type: apps.RollingUpdateStatefulSetStrategyType,
+				Type: baseWorkerSpec.StatefulSetUpdateStrategy(),
```

Comment:
No need to consider the failover case here as we do for the dm-master?

Reply:
for dm-worker, is it no need to use `.spec.updateStrategy.rollingUpdate.partition` feature? I don't know why and `partition` was never added for dm-worker before. and Pump without `partition` too.

Reply:
OK, I understand, there is no extra operation to do for dm-worker and Pump.